data_management/movie.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os
+from imdb import IMDb
+from pymongo import MongoClient
+
+
+DB_URL = "mongo.fettay.com"
+DB_NAME = 'recommander'
+DB_COLLECTION = 'movies'
+
+
+class Movie:
+    def __init__(self, imdbID: str, title: str = '', votes: int = -1, genres: list = [], \
+        plot: list = [], synopsis: str = '', keywords: list = [], tagline: str = ''):
+        self.imdbID = imdbID
+        self.title = title
+        self.votes = votes
+        self.genres = genres
+        self.plot = plot
+        self.synopsis = synopsis
+        self.keywords = keywords
+        self.tagline = tagline
+
+
+    def load_from_imdb(self):
```

Comment:
This method should be a static method and take a movie_id as argument and return an instance of Movie: when you use it the code should look like this:
`movie = Movie.load_from_imdb(movie_id)`
Instead of:
`movie = Movie(movie_id)`
`movie.load_from_imdb()`